ibm/provider.go

```diff
@@ -154,117 +154,118 @@ func Provider() terraform.ResourceProvider {
+			"ibm_certificate_manager_certificates":       dataIBMCertificateManagerCertificates(),
+			"ibm_certificate_manager_certificate":        dataIBMCertificateManagerCertificate(),
+			"ibm_cis":                                    dataSourceIBMCISInstance(),
+			"ibm_cis_dns_records":                        dataSourceIBMCISDNSRecords(),
+			"ibm_cis_global_load_balancers":              dataSourceIBMCISGlbs(),
+			"ibm_cis_origin_pools":                       dataSourceIBMCISOriginPools(),
+			"ibm_cis_healthchecks":                       dataSourceIBMCISHealthChecks(),
+			"ibm_cis_domain":                             dataSourceIBMCISDomain(),
+			"ibm_cis_firewall":                           dataIBMCISFirewallsRecord(),
+			"ibm_cis_waf_packages":                       dataSourceIBMCISWAFPackages(),
+			"ibm_cis_rate_limit":                         dataSourceIBMCISRateLimit(),
+			"ibm_cis_ip_addresses":                       dataSourceIBMCISIP(),
+			"ibm_cis_edge_functions_actions":             dataSourceIBMCISEdgeFunctionsActions(),
+			"ibm_cis_edge_functions_triggers":            dataSourceIBMCISEdgeFunctionsTriggers(),
+			"ibm_cis_custom_pages":                       dataSourceIBMCISCustomPages(),
+			"ibm_cis_page_rules":                         dataSourceIBMCISPageRules(),
+			"ibm_database":                               dataSourceIBMDatabaseInstance(),
+			"ibm_compute_bare_metal":                     dataSourceIBMComputeBareMetal(),
+			"ibm_compute_image_template":                 dataSourceIBMComputeImageTemplate(),
+			"ibm_compute_placement_group":                dataSourceIBMComputePlacementGroup(),
+			"ibm_compute_ssh_key":                        dataSourceIBMComputeSSHKey(),
+			"ibm_compute_vm_instance":                    dataSourceIBMComputeVmInstance(),
+			"ibm_container_addons":                       datasourceIBMContainerAddOns(),
+			"ibm_container_alb":                          dataSourceIBMContainerALB(),
+			"ibm_container_alb_cert":                     dataSourceIBMContainerALBCert(),
+			"ibm_container_bind_service":                 dataSourceIBMContainerBindService(),
+			"ibm_container_cluster":                      dataSourceIBMContainerCluster(),
+			"ibm_container_cluster_config":               dataSourceIBMContainerClusterConfig(),
+			"ibm_container_cluster_versions":             dataSourceIBMContainerClusterVersions(),
+			"ibm_container_cluster_worker":               dataSourceIBMContainerClusterWorker(),
+			"ibm_container_vpc_cluster_alb":              dataSourceIBMContainerVPCClusterALB(),
+			"ibm_container_vpc_alb":                      dataSourceIBMContainerVPCClusterALB(),
+			"ibm_container_vpc_cluster":                  dataSourceIBMContainerVPCCluster(),
+			"ibm_container_vpc_cluster_worker":           dataSourceIBMContainerVPCClusterWorker(),
+			"ibm_container_vpc_worker_volume_attachment": dataSourceIBMContainerVpcWorkerVolumeAttachment(),
```

Comment:
can we rename ibm_container_vpc_worker_volume_attachment to ibm_container_vpc_worker_storage